lucene/queries/src/java/org/apache/lucene/queries/intervals/IntervalIterator.java

```diff
@@ -82,6 +82,11 @@ public int width() {
   /**
    * Advance the iterator to the next interval
    *
+   * Should not be called after {@link DocIdSetIterator#NO_MORE_DOCS} is returned by other methods
```

Comment:
Hmm maybe be more specific than `other methods`?  E.g. maybe say `returned by the query scorer's nextDoc() method`?